boards/arm/arduino_portenta_h7/arduino_portenta_h7_m7.yaml

```diff
@@ -0,0 +1,17 @@
+identifier: arduino_portenta_h7_m7
+name: Arduino Portenta H7
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+ram: 512
+flash: 1024
+supported:
+  - arduino_gpio
+  - gpio
+  - arduino_spi
```

Comment:
I am not sure what this line means but maybe an `arduino_i2c` could also be added?

And, should not these lines added in the dts?
```
arduino_spi: &spi2 {};
arduino_i2c: &i2c3 {};
arduino_serial: &usart1 {};
```

Reply:
This should be removed as arduino Uno connector is not available.
Same for `arduino_gpio`